lib/action_view/component/template_error.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ActionView
+  module Component
+    class TemplateError < StandardError
+      def initialize(errors)
+        super(errors.join(", "))
+      end
+    end
+  end
+end
```

Comment:
I added a our own Error. We could maybe do better than concating the errors but it probably does for now.